src/pages/HealthContent.tsx

```diff
@@ -1,10 +1,133 @@
 import React from "react";
+// @ts-ignore
+import workoutImg from "../assests/images/workout.jpg";
 import styles from "./HealthContent.module.css";
```

Comment:
I'd recommend always making CSS imports last. That way, if you import a library that comes with CSS, you can override the styling since your CSS comes after it.

Reply:
Thanks for the tip. I would sometimes see developers do that, but I didn't know the reason